src/utils/runbookFilename.ts

```diff
@@ -0,0 +1,47 @@
+import * as vscode from "vscode";
+
+export const generateFriendlyPromptName = (promptType: string) => {
+    let friendlyPromptName = promptType;
+
+    if (friendlyPromptName.startsWith("github:")) {
+        const repo = friendlyPromptName.split("github:")[1].split("?")[0].replace("/", "-");
+        const path = friendlyPromptName.split("path=")[1].split("&")[0].replaceAll("/", "-");
+        friendlyPromptName = `gh-${repo}:${path}`;
```

Comment:
Will this be a dangling `:` when the path is not present. The repo is mandatory but the path is not.

Reply:
Also windows didn't used to support colons in filenames and certain programs on macos, like Finder, were confused by them too. I don't think we should use them.